File: Selenium/Project/Activity6.py
# Import necessary packages
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the driver using with statement
with webdriver.Firefox() as driver:
    # Open the page
    driver.get("http://alchemy.hguy.co/crm")
    
    # Find username and password fields
    username_field = driver.find_element(By.ID, "user_name")
    password_field = driver.find_element(By.ID, "username_password")
    login_button = driver.find_element(By.ID, "bigbutton")
    
    # Enter credentials
    username_field.send_keys("admin")
    password_field.send_keys("pa$$w0rd")
    
    # Click the submit button
    login_button.click()
    
    # Find the Activities menu
    activities_menu = driver.find_element(By.ID, "grouptab_3")
    
    # Check if the Activities menu is displayed and click it
    if activities_menu.is_displayed():
        activities_menu.click()
        print("Activities menu is clicked.")

File: Selenium/Project/Activity6.java
package FST_Selenium_Project;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity6 {
    WebDriver driver;
	
@BeforeClass
public void setUp() {
	driver = new FirefoxDriver();
	driver.get("http://alchemy.hguy.co/crm");
	WebElement userName = driver.findElement(By.id("user_name"));
	WebElement password = driver.findElement(By.id("username_password"));
	WebElement login = driver.findElement(By.id("bigbutton"));
	
	userName.sendKeys("admin");
	password.sendKeys("pa$$w0rd");
	login.click();
}
 @Test
 public void activitiesMenu() {
	 WebElement activitiesMenu = driver.findElement(By.id("grouptab_3"));
	 if(activitiesMenu.isDisplayed()) {
		 activitiesMenu.click();
		 System.out.println("is clicked: ");
	 }
	 
	 
 }
 
 
 
@AfterClass
public void tearDown() {
	driver.quit();
}
}
